packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx

```diff
@@ -360,6 +360,7 @@ function mapStatetoProps(
         ...draft.data,
         template: {
           ...draft.data.template,
+          informantType: draft.data.informant.relationship,
```

Comment:
Hey! To do this as per the convention, this should be done in `default/birth.ts` inside the mapping block here:
https://github.com/opencrvs/opencrvs-core/blob/fd52441d619d69319d775006bae52ce0799390db/packages/client/src/forms/configuration/default/birth.ts#L121

Adding
```
template: {
  fieldName: 'informantType',
  operation: 'plainInputTransformer'
}
```

might even do the trick

Reply:
I have just applied your recommendation but I have not had the same result.